consensus/istanbul/backend/backend.go

```diff
@@ -442,7 +438,7 @@ func (sb *Backend) Close() error {
 	if err := sb.valEnodeTable.Close(); err != nil {
 		errs = append(errs, err)
 	}
-	if err := sb.versionCertificateTable.Close(); err != nil {
+	if err := sb.announceManager.Close(); err != nil {
```

Comment:
not of this PR, but i'm sure there's some library of code example on how to actually group errors. At least using %w on Errof, but doing that would imply (err1 -> err2 -> err3); when is actually (err -> [err1, err2, err3])

Reply:
article about wrapping errors https://blog.golang.org/go1.13-errors

Reply:
+1 We can / should do those